assets/src/components/v2/persistent_wrapper.tsx

```diff
@@ -50,7 +35,7 @@ const PersistentWrapper: ComponentType<Props> = ({
  * defining your `WrappedComponent`'s prop types.
  *
  * `WrappedComponent` must expect the following props:
- * - `onFinish`: A callback that `WrappedComponent` can call to indicate that it is ready to receive
+ * - `updateVisibleData`: A callback that `WrappedComponent` can call to indicate that it is ready to receive
```

Comment:
Seems like we could revisit the wording of this documentation since it's not really specific to data refreshes anymore.

Reply:
Updated the documentation here, but not sure if I love my updated wording 

```
 * - `updateVisibleData`: A callback that `WrappedComponent` can call to refresh the data displayed by the child component
 *   with the most recently received data.
 * - `lastUpdate`: a **nullable** timestamp of the last successful data refresh, which `WrappedComponent` should use to
 *   sync its re-renders with the rest of the page. Used by components that use data refreshes to handle pagination.
```